terraform/pilot/main.tf

```diff
@@ -15,15 +15,20 @@ variable "branch" {}
 
 variable "security_context" { default = "dev" }
 
-variable "aws_region" { default = "us-east-1" }
-variable "ami_type" { default = "hardened" }
+variable "network" { 
+    type = "map"
+    default = {
+        region = "us-east-1" 
+    }
+}
+variable "ami_type" { default = "not-hardened" }
 variable "bastion_subnet_id" {}
 variable "bastion_security_group_id" {}
-variable "bastion_instance_type" {}
+variable "bastion_instance_type" { default = "m3.medium" }
```

Comment:
generally speaking for our compute heavy applications c4.xlarge or c4.2xlarge for mem heavy m4.xlarge/m4.2xlarge for prod loads for demo/dev **c4.large and m4.large**